### PR TITLE
[BACKLOG-33305]Snowflake - Add write support for ORC and Parquet on P…

### PR DESCRIPTION
**Pull request by tkafalas**
…VFS connections

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 50s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl kettle-plugins/formats
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: kettle-plugins/formats/src/main/java/org/pentaho/big/data/kettle/plugins/formats/impl/parquet/output/ParquetOutput.java [hunk 1]
```diff
@@ -171,4 +197,25 @@ public boolean init( StepMetaInterface smi, StepDataInterface sdi ) {
     }
     return true;
   }
+
+  private void copyFileAndDeleteTempSource( ) throws KettleFileException, IOException {
```
Comment (mkambol) [inline kettle-plugins/formats/src/main/java/org/pentaho/big/data/kettle/plugins/formats/impl/parquet/output/ParquetOutput.java:201]: Does the Snowflake VFS not already delete the temp file on completion?  If not, seems like it should-- I wouldn't think each consumer of the vfs impl should be responsible.

Comment (tkafalas) [reply kettle-plugins/formats/src/main/java/org/pentaho/big/data/kettle/plugins/formats/impl/parquet/output/ParquetOutput.java:201]: Yes, it was not implemented at all other than to throw an unsupported operation

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 23s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl kettle-plugins/formats
```
---
:ok_hand: All tests passed!
 
Tests run: **40**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-plugin/job/PR-2050/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**